lti_consumer/utils.py

```diff
@@ -64,3 +64,15 @@ def get_lms_lti_access_token_link(location):
         lms_base=get_lms_base(),
         location=str(location),
     )
+
+
+def get_lti_ags_lineitems_url(lti_config_id):
+    """
+    Return the LTI AGS endpoint
+
+    :param lti_config_id: LTI configuration id
+    """
+    return u"{lms_base}/api/lti_consumer/v1/lti/{lti_config_id}/lti-ags".format(
```

Comment:
Probably `u` string is redundant as all python strings are Unicode in python 3. This block seems to only support Python 3 according to [setup.py](https://github.com/edx/xblock-lti-consumer/blob/master/setup.py)

Reply:
Yup, I've fixed it and references in the same file.